.rubocop_1.x.yml

```
@@ -0,0 +1,496 @@
+# If a cop is not referenced here, the defaults from Rubocop will be used.
+
+AllCops:
+  Exclude:
+    - '**/db/**/*'
+    - '**/tmp/**/*'
+    - '**/vendor/**/*'
+    - '**/script/**/*'
+  DisplayStyleGuide: true
+  ExtraDetails: true
+  UseCache: true
+  MaxFilesInCache: 50000
+  TargetRubyVersion: 2.6
```

Some SOP apps are still on Ruby 2.4 e.g. the MS1s, so it'd probably be prudent to use 2.4 here:
```suggestion
  TargetRubyVersion: 2.4
```

---

Yeah - I would agree - we can override this on a per-repo basis anyway

---

fixed :+1: 

note (to myself for the future) that it will then lock the version of rubocop running in .codeclimate.yml

```
engines:
  rubocop:
    enabled: true
    channel: rubocop-1-12-1
```